src/app/shared/help-support/help-support.html

```diff
@@ -0,0 +1,25 @@
+<div class="docs-help-support">
+  <ul>
+    <li>
+      <a href="https://stackoverflow.com/questions/tagged/angular-material" target="_blank">
+        <img src="../../../assets/img/help-support/stackoverflow-logo.svg" alt="stack-overflow">
+        StackOverflow
+      </a>
+      <div class="docs-help-support-description">The latest issue on the StackOverflow.</div>
+    </li>
+    <li>
+      <a href="https://gitter.im/angular/material2" target="_blank">
+        <img src="../../../assets/img/help-support/gitter-logo.svg" alt="gitter">
+        Gitter
+      </a>
+      <div class="docs-help-support-description">Discussion group about Angular Material on Gitter</div>
+    </li>
+    <li>
+      <a href="https://groups.google.com/g/angular-material2" target="_blank">
```

Comment:
```suggestion
      <a href="https://groups.google.com/g/angular-material2">
```
We don't use `target-"_blank"` anywhere on the site and using it without `rel="noopener"` causes Lighthouse Best Practice audit issues.